kdtree.py

```diff
@@ -219,7 +219,8 @@ def __init__(self, data=None, left=None, right=None, axis=None,
         sel_axis(axis) is used when creating subnodes of the current node. It
         receives the axis of the parent node and returns the axis of the child
         node. """
-
+        if type(data) == list:
```

Comment:
Only a type check, to ensure no error while copying this list.
Maybe it's unnecessary (because this function should only access list as argument), I can remove it.